internal/server/root_certificate.go

```diff
@@ -5,6 +5,8 @@ import (
 	"fmt"
 
 	"github.com/hashicorp/boundary/internal/errors"
+	wrapping "github.com/hashicorp/go-kms-wrapping/v2"
+	"github.com/hashicorp/go-kms-wrapping/v2/extras/structwrapping"
 
 	timestamp "github.com/hashicorp/boundary/internal/db/timestamp"
```

Comment:
Not your fault but I think we should merge these import groups, we generally only have two.